src/test/java/kg/jarkyn/server/TCPServerTest.java

```diff
@@ -0,0 +1,12 @@
+package kg.jarkyn.server;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class TCPServerTest {
+    @Test
+    public void noIdeaWhatAmITestingHere() throws IOException {
```

Comment:
this is to give you guys an idea that i have no idea what to do here
(working on this part right nao)